### DISCUSSION TITLE
[NPC 생성] 첫 월드 접속 이후 재접속 시 NPC 생성 안됨

### DISCUSSION THREAD

**Discussion by Joyounwoo** [naverz/zepeto-studio-kor#699] [Issue Report (Legacy)]
NPC를 생성할 때 다음과 같은 코드를 사용하고 있습니다.

`ZepetoPlayers.instance.CreatePlayerWithUserId("NPC", userId, spawnInfo, false);`

테스트링크를 통해 첫 월드 접속 시 NPC의 생성이 정상적으로 동작합니다. 하지만 그 이후 월드 접속 시에는 NPC가 존재하지 않게 됩니다.
제페토를 종료하고 다시 실행해서 테스트해도 동일한 문제가 발생합니다.
하지만 일정 시간 이후 재접속 했을 때 정상적으로 나타납니다.

유니티 에디터 접속 시에는 정상 동작하지만 테스트 링크를 통한 제페토 앱에서는 위와 같은 문제가 있습니다.
`CreatePlayerWithUserId`의 사용법이 의도하셨던 바와 다를까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요,  ZepetoPlayer는  다음과 같이 두 가지 형태가 존재합니다.

- Local Player: 로컬 유저가 직접 조작 가능한 형태의 캐릭터 인스턴스입니다(Controller/Camera 컴포넌트가 부착되어 있음).
- Player: Multiplay 컨텐츠에 로드하여 활용할 수 있는 캐릭터 인스턴스입니다(Controller/Camera 컴포넌트가 부착되지 않음).

NPC는 ZepetoCharacter를 사용해서 생성해 보세요.

https://studio.zepeto.me/guides/zepetocharacter

**Reply by Joyounwoo** [reply]
이슈에 버그 리포트 예시 글 보았습니다. 먼저 읽어보고 올릴 걸 그랬네요!
